src/app/jokes/jokes.component.html

```diff
@@ -1 +1,19 @@
-<p>jokes works!</p>
+<div class="container">
+    <button (click)="displayJoke()">Joke</button>
+    <p>{{jokes.value}}</p>
+
+    <form [formGroup]="form" (ngSubmit)="onSubmit(form)">
+        <div class="input-group">
+            <select class="form-select" formControlName = "selectOption" id="inputGroupSelect04" aria-label="Example select with button addon">
+              <option *ngFor="let category of categories">{{category}}</option>
```

Comment:
when you use select component within a form, value attribute on option is what gets used as a control value